### PR TITLE
Add PyCodeQual.

### PR DESCRIPTION
**Pull request by moser**
Adding PyCodeQual, my SaaS product :-)

### CODE REVIEW

File: data/tools/pycodequal.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: PyCodeQual
+categories:
+  - linter
+tags:
+  - python
+license: proprietary
+types:
+  - service
+homepage: 'https://pycodequ.al'
+description: Static Python code analysis
```
Comment (mre) [inline data/tools/pycodequal.yml:10]: Hum, the description is rather generic right now. Do you think you can highlight a few key features that sets PyCodeQual apart from other services?
The [full list is here](https://analysis-tools.dev/tag/python) for reference.

**Comment by mre** [comment]
Thanks for the PR @moser. Just added an idea for improvement. Let me know what you think.

**Comment by moser** [comment]
Hey. I extended the description a bit :-)

**Comment by mre** [comment]
Thanks, looks good. 😊 Merging.

**Comment by mre** [comment]
Cheers from a fellow Upper Palatinate. 🍷 😉

**Pull request merged**